www/Instabug.js

```diff
@@ -273,8 +273,15 @@ Instabug.enable = function (success, error) {
     exec(success, error, 'IBGPlugin', 'enable', []);
 };
 
+/**
+ * Gets a boolean indicating whether the SDK is enabled or not
+ * It works on android only
+ * 
+ * @param {function(boolean):void} success callback on function success
+ * @param {function(string):void} error callback on function error
+ */
 Instabug.isEnabled = function (success, error) {
```

Comment:
If this has been broken for a while, I suggest adding a suffix to indicate it only works for Android. What do you think?